ezResx/Excel/ExcelReader.cs

```diff
@@ -139,17 +140,17 @@ private void VerifyStringFormat(List<ResourceItem> resources)
                     var count = regex.Matches(resourceValue.Value).Count;
                     if (count != stringFormatCount)
                     {
-                        exceptions.Add(
-                            new Exception(
-                                $"String format missmatch in {resource.Key.Project} {resource.Key.File} {resource.Key.Name}"));
+                        lostData.Add(resource);
```

Comment:
Probably could go with continue; to avoid multiple repetitions of the same key in the output